raiden/ui/app.py

```diff
@@ -229,6 +230,7 @@ def run_app(
     config['transport']['udp']['nat_keepalive_timeout'] = timeout
     config['privatekey_hex'] = encode_hex(privatekey_bin)
     config['unrecoverable_error_should_crash'] = unrecoverable_error_should_crash
+    config['services'] = {'pathfinding-service-address': pathfinding_service_address}
```

Comment:
Looking at the other keys, this shoudl probably use underscores instead of dashes.

Reply:
Never!

Reply:
Also a default for this value (even if it's `None`) should be added to `raiden/app.py::App.DEFAULT_CONFIG`

Reply:
done

Reply:
The default is still missing.